### PR TITLE
More edits

### PR DESCRIPTION
**Pull request by landodragon141**
Edits to Motor Movement, Axis Homing, and Filament Run out sensor.

### CODE REVIEW

File: _development/configuration.md [hunk 1]
```diff
@@ -502,7 +504,37 @@ Tip: 0.02 mm is normally acceptable for bed leveling to work.
...
+{% endhighlight %}
+
+{% highlight stepper_disable %}
+#define DISABLE_X false
+#define DISABLE_Y false
+#define DISABLE_Z false
+{% endhighlight %}
+
+This enabling this value will disable the given stepper when it is not in use. In most situations you should leave the default setting. The reason you would want this is you are running your steppers at higher than normal currents in order to get extra torque from them. By disabling the steppers in between moves it will allow them to run cooler. That sounds good but there's a catch. Because the steppers are disabled they will not be holding the axis stable. This WILL result in bad accuracy and carries a strong posibility of axis drift. Remember most 3d printers use open loop control systems. This means that the software has no idea what the actual position of the axis is. In practice with a well calibrated machine this is not an issue and using an open loop is one of the major cost savings. To be honest I can't really recommend this in any situation as the cons outweigh the pros. There are better solutions to this problem: stepper motor heatsink, dual motors on the axis, active cooling, etc.
```
Comment (thinkyhead) [inline _development/configuration.md]: "This enabling this value"?

Comment (thinkyhead) [inline _development/configuration.md]: I'll try to put together a "style guide" soon. We need to write in a more neutral and less familiar style, eliminate use of the word "you" and "your."

**Comment by landodragon141** [comment]
This has been edited.

**Comment by landodragon141** [comment]
Yo Scott I fixed it. :-D

**Comment by thinkyhead** [comment]
Good stuff. I hope to write up a couple of GCode commands each day, and hopefully this will be completed by the time 1.1.0 is out.

**Comment by landodragon141** [comment]
I saw what you just wrote it looked great.

**Pull request merged**